### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by nimrodkor**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ termcolor="*"
 junit-xml ="*"
 dpath = ">=1.5.0,<2"
 pyyaml = ">=5.4.1"
-boto3 = "*"
+boto3 = "==1.17.27"
```
Comment (robeden) [inline Pipfile:29]: Curious: why does this improve performance?

Comment (schosterbarak) [reply Pipfile:29]: we are not sure. trial and error. :(

Comment (nimrodkor) [reply Pipfile:29]: That's an excellent question. Still trying to figure that one out

Comment (schosterbarak) [reply Pipfile:29]: for me today felt like the commits at: https://twitter.com/gitlost

**Pull request merged**